frontend/app/src/app/logs/page.tsx

```
@@ -0,0 +1,196 @@
+      <div>
+        <label htmlFor="integration">Integration Name</label>
+        <select
+          name="integration"
+          id="integration"
+          value={integration}
+          onChange={(e) => setIntegration(e.target.value)}
+          className="w-full rounded-sm bg-whiten p-2 text-black focus:outline-none focus:ring-1 focus:ring-blue-500 dark:bg-gray-700 dark:text-white"
+        >
+          <option value="-">----</option>
+          {integrations?.data.map((pipeline: any) => (
+            <option value={pipeline.id}>{pipeline.integration_name}</option>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)